contracts/issuer/IssuerAccessFacet.sol

```diff
@@ -72,4 +73,47 @@ contract IssuerAccessFacet is AIssuerFacet, IHasMembers {
     // Emit!
     emit MemberRemoved(member);
   }
+
+  /** @notice Callback from FAST contracts allowing the Issuer contract to keep track of governorships.
+   * @param governor The governor added to a FAST.
+   */
+  function governorAddedToFast(address governor)
+      external {
+    // Verify that the given address is in fact a registered FAST contract.
+    require(
+      IssuerTopFacet(address(this)).isFastRegistered(msg.sender),
+      LibConstants.REQUIRES_FAST_CONTRACT_CALLER
+    );
+    // Keep track of the governorship.
+    LibAddressSet.Data storage governorships = LibIssuerAccess.data().fastGovernorships[governor];
```

Comment:
Super small saving, but we could skip this operation and go directly:
```solidity
LibIssuerAccess.data().fastGovernorships[governor].add(msg.sender, false);
``` 
Same for the `governorRemovedFromFast()`

Reply:
It's a fair point @afmfe-iul

Reply:
@jamesduncombe I don't like this kind of optimisation in standard code because it hurts readability. But in Solidity, @afmfe-iul is right.
The rule of thumb is:
- If the storage is only used once, inline the call to the storage pointer getter.
- If it's used more than once, do it like you did.
So in this case, let's follow @afmfe-iul's advive :)